Windows10DebloaterGUI.ps1

```diff
@@ -210,152 +210,270 @@ $global:WhiteListedAppsRegex = $global:WhiteListedApps -join '|'
-$DisableTelemetry.Height = 35
-$DisableTelemetry.Location = New-Object System.Drawing.Point(365, 260)
-$DisableTelemetry.Font = 'Microsoft Sans Serif,10'
-
-$RemoveRegkeys = New-Object System.Windows.Forms.Button
-$RemoveRegkeys.Text = "Remove Bloatware Regkeys"
-$RemoveRegkeys.Width = 188
-$RemoveRegkeys.Height = 35
-$RemoveRegkeys.Location = New-Object System.Drawing.Point(540, 260)
-$RemoveRegkeys.Font = 'Microsoft Sans Serif,10'
-
-$UnpinStartMenuTiles = New-Object System.Windows.Forms.Button
-$UnpinStartMenuTiles.Text = "Unpin Tiles From Start Menu"
-$UnpinStartMenuTiles.Width = 190
-$UnpinStartMenuTiles.Height = 35
-$UnpinStartMenuTiles.Location = New-Object System.Drawing.Point(540, 217)
-$UnpinStartMenuTiles.Font = 'Microsoft Sans Serif,10'
-
-$RemoveOnedrive = New-Object System.Windows.Forms.Button
-$RemoveOnedrive.Text = "Uninstall OneDrive"
-$RemoveOnedrive.Width = 152
-$RemoveOnedrive.Height = 35
-$RemoveOnedrive.Location = New-Object System.Drawing.Point(365, 217)
-$RemoveOnedrive.Font = 'Microsoft Sans Serif,10'
-
-#$FixWhitelist = New-Object System.Windows.Forms.Button
-#$FixWhitelist.Text = "Fix Whitelisted Apps"
-#$FixWhitelist.Width = 130
-#$FixWhitelist.Height = 37
-#$FixWhitelist.Location = New-Object System.Drawing.Point(254, 74)
-#$FixWhitelist.Font = 'Microsoft Sans Serif,10'
-
-$InstallNet35 = New-Object System.Windows.Forms.Button
-$InstallNet35.Text = "Install .NET v3.5"
-$InstallNet35.Width = 152
-$InstallNet35.Height = 39
-$InstallNet35.Location = New-Object System.Drawing.Point(169, 335)
-$InstallNet35.Font = 'Microsoft Sans Serif,10'
-
-$EnableDarkMode = New-Object System.Windows.Forms.Button
-$EnableDarkMode.Text = "Enable Dark Mode"
-$EnableDarkMode.Width = 152
-$EnableDarkMode.Height = 39
-$EnableDarkMode.Location = New-Object System.Drawing.Point(9, 335)
-$EnableDarkMode.Font = 'Microsoft Sans Serif,10'
-
-$DisableDarkMode = New-Object System.Windows.Forms.Button
-$DisableDarkMode.Text = "Disable Dark Mode"
-$DisableDarkMode.Width = 152
-$DisableDarkMode.Height = 39
-$DisableDarkMode.Location = New-Object System.Drawing.Point(9, 385)
-$DisableDarkMode.Font = 'Microsoft Sans Serif,10'
-
-
-
-$Form.controls.AddRange(@($Debloat, $CustomizeBlacklists, $RemoveAllBloatware, $RemoveBlacklist, $Label1, $RevertChange, $Label2, $DisableCortana, $EnableCortana, $StopEdgePDFTakeover, $EnableEdgePDFTakeover, $DisableTelemetry, $RemoveRegkeys, $UnpinStartMenuTiles, $RemoveOnedrive, $FixWhitelist, $RemoveBloatNoBlacklist, $InstallNet35, $EnableDarkMode, $DisableDarkMode))
+$Form                            = New-Object system.Windows.Forms.Form
+$Form.ClientSize                 = New-Object System.Drawing.Point(500,570)
+$Form.StartPosition              = 'CenterScreen'
+$Form.FormBorderStyle            = 'FixedSingle'
+$Form.MinimizeBox                = $false
+$Form.MaximizeBox                = $false
+$Form.ShowIcon                   = $false
+$Form.text                       = "Windows10Debloader"
```

Comment:
I think it's not 'debloader'. It should be 'debloater'.

Reply:
Yes you're right, i'll modify this quickly !